.concourse/post-publish.yaml

```diff
@@ -0,0 +1,346 @@
+
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: teliaoss/github-pr-resource
+- name: github-status
+  type: docker-image
+  source:
+    repository: resource/github-status
+    tag: release
+resources:
+  - name: kubecf
+    type: git
+    source:
+      branch: master
+      uri: https://github.com/SUSE/kubecf
+  - name: catapult
+    type: git
+    source:
+      uri: https://github.com/SUSE/catapult
+  - name: status.src
+    type: github-status
+    source:
+      repo: SUSE/kubecf
+      access_token: ((github-access-token))
+
+deploy_args: &deploy_args
+  - -xce
+  - |
+    export SCF_LOCAL=$PWD/kubecf
+    pushd catapult
+    export SCF_OPERATOR=true
+    export FORCE_DELETE=true
+    export SCF_TESTGROUP=true
+    export BACKEND=ekcp
+    export DOCKER_ORG=cap-staging
+    export QUIET_OUTPUT=true
+
+    # Bring up a k8s cluster and builds+deploy kubecf
+    # See: https://github.com/SUSE/catapult/wiki/Build-and-run-SCF#build-and-run-kubecf
+    make k8s scf-build
+
+test_args: &test_args
+  - -xce
+  - |
+    export BACKEND=ekcp
+    export KUBECF_TEST_SUITE="${TEST_SUITE:-smokes}"
+    export KUBECF_NAMESPACE="scf"
+    export KUBECF_CHECKOUT="$PWD/kubecf"
+    export QUIET_OUTPUT=true
+    pushd catapult
+
+    # Grabs back a deployed cluster and runs test suites on it
+    # See: https://github.com/SUSE/catapult/wiki/Running-SCF-tests#kubecf
+    make recover tests-kubecf
+
+jobs:
+
+# Diego
+  - name: deploy-kubecf-diego-master
```

Comment:
I don't think we need to specify the branch name as a postfix since the pipeline is tied to a specific branch already.